db/migrate/20170116210702_add_manger_ref_to_authentications.rb

```diff
@@ -0,0 +1,5 @@
+class AddMangerRefToAuthentications < ActiveRecord::Migration[5.0]
+  def change
+    add_column :authentications, :manager_ref, :string
```

Comment:
I'm amazed this was never added.   How does Amazon refresh even work o_O  cc @blomquisg @Ladas